mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -78,6 +81,16 @@ private object DTO {
       totalDeletedSize,
       mailboxId)
   }
+
+  case class Added(sessionId: SessionId, user: User, path: MailboxPath, mailboxId: MailboxId,
+                   added: Map[MessageUid, MetaDataDTO.MessageMetaData]) extends Event {
+    override def toJava: JavaEvent = new JavaAdded(
+      sessionId,
+      user,
+      path.toJava,
+      mailboxId,
+      added.map(entry => entry._1 -> entry._2.toJava).asJava)
```

Comment:
this looks not good, I tried to move on to have a worked implementation first

Reply:
I'm not shocked much ^^